test_natsort/test_natsort_cmp.py

```diff
@@ -13,7 +14,8 @@
 
 
 if PY_VERSION >= 3.0:
-    cmp = lambda a, b: (a > b) - (a < b)
+    def cmp(a, b):
```

Comment:
I would rather see `cmp` defined in [`natsort.compat.py23.py`](https://github.com/SethMMorton/natsort/blob/master/natsort/compat/py23.py), because then you can import that definition both in the code and in the tests. It will be a bit cleaner that way.

Reply:
I agree